distributed/cli/utils.py

```diff
@@ -69,3 +71,12 @@ def uri_from_host_port(host_arg, port_arg, default_port):
     addr = unparse_address(scheme, loc)
 
     return addr
+
+
+def create_ssl_context(certfile, keyfile):
+    if certfile and keyfile:
+        ssl_ctx = ssl.create_default_context(ssl.Purpose.SERVER_AUTH)
```

Comment:
Chose this since it seemed like pretty much all dask things were both clients and servers and it seemed to work at least.  Given that the `ssl.Purpose` was reintroduced into python 3.4 I'm unclear how much this matters.